appcelerator/titanium_mobile
# fix(android): update log strings TIMOB-7786

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-7786

**Summary:**
Fixing typos in Log.

---
**Test:**
1. Build and run the below on Android.
2. Verify that the exception message reads "decodeNumber", not "encodeNumber", as shown below.
`Error: source was not specified for decodeNumber`

---
app.js
```javascript
var encodedValue = 123456;
var buffer = Ti.createBuffer({ length: 4 });
Ti.Codec.encodeNumber({
	source: encodedValue,
	dest: buffer,
	type: Ti.Codec.TYPE_INT,
})
var decodedValue = Ti.Codec.decodeNumber({
//	source: buffer,
	type: Ti.Codec.TYPE_INT,
});
alert("Encoded Value: " + encodedValue + "\nDecoded Value: " + decodedValue);
```
